.bazelrc

```diff
@@ -298,6 +298,7 @@ build:windows --action_env=TMPDIR
 build:windows --define signal_trace=disabled
 build:windows --define hot_restart=disabled
 build:windows --define tcmalloc=disabled
+build:windows --define wasm=disabled
```

Comment:
I don't object to this change, but it seems unrelated to the main topic of this PR. Can it be in a separate PR?

Reply:
The reason it is baked into this PR is because you can't build `//source/exe:envoy-static` without this definition so it is part of my inner dev loop.

No strong opinions though.

Reply:
It's probably fine; I don't feel strongly about it.